plugins/acousticbrainz_tonal-rhythm/acousticbrainz_tonal-rhythm.py

```diff
@@ -44,26 +45,38 @@
 class AcousticBrainz_Key:
 
     def get_data(self, album, track_metadata, trackXmlNode, releaseXmlNode):
+        if not musicbrainz_recordingid in track_metadata:            
+            log.error("%s: Error parsing response. No musicbrainz recording id found.",
+                      PLUGIN_NAME)
+            return
         recordingId = track_metadata['musicbrainz_recordingid']
         if recordingId:
-            log.debug("%s: Add AcusticBrainz request for %s (%s)", PLUGIN_NAME, track_metadata['title'], recordingId)
+            log.debug("%s: Add AcousticBrainz request for %s (%s)",
+                      PLUGIN_NAME, track_metadata['title'], recordingId)
             self.album_add_request(album)
             path = "/%s/low-level" % recordingId
             return album.tagger.webservice.get(
                         ACOUSTICBRAINZ_HOST,
                         ACOUSTICBRAINZ_PORT,
                         path,
                         partial(self.process_data, album, track_metadata),
-                        priority=True, important=False)
-        return
+                        priority=True,
+                        important=False,
+                        parse_response_type=None)
 
     def process_data(self, album, track_metadata, response, reply, error):
         if error:
             log.error("%s: Network error retrieving acousticBrainz data for recordingId %s",
-                PLUGIN_NAME, track_metadata['musicbrainz_recordingid'])
+                      PLUGIN_NAME, track_metadata['musicbrainz_recordingid'])
+            self.album_remove_request(album)
+            return
+        try:
+            data = parse_json(response)
+        except JSONDecodeError:
+            log.error("%s: Network error retrieving acousticBrainz data for recordingId %s",
```

Comment:
Extremely minor nitpick, but: "acousticBrainz" should be "AcousticBrainz". :)
```suggestion
            log.error("%s: Network error retrieving AcousticBrainz data for recordingId %s",
```

Reply:
Fixed.